### ISSUE TITLE
accessibility problem in travis web ui with switches/checkboxes

### ISSUE THREAD

**Issue by webczat** [travis-ci/travis-ci#7012]
Hello.

There is one quite small accessibility problem on travis web ui.
That is, the checkboxes are not accessible. Like those to enable building pull requests, or those to activate travis for a given repository.
The only way I can toggle this is by clicking using mouse emulation of a screenreader, those switches are not focusable by keyboard and cannot be toggled using it.

Fixing it would make it nicer.

**Comment by timofonic** [comment]
@BanzaiMan @webczat I'm handicapped in certain way, because I'm a keyboard junkie and awfully bad with pointing devices fastly. 

I also sometimes suffer pain in my wrists and hands when using bad pointing devices or keyboards too much, but it got totally reduced by a lot less with the ignored but amazing[ trackballs](http://clearlysuperiortech.com/cst2545-5wglredproductpage.html) and a mechanical keyboard.

I believe those are very early signs of [Carpal Tunnel Syndrome](https://en.wikipedia.org/wiki/Carpal_tunnel_syndrome). Many doctors and fitness trainers said me I have very flexible wrists and I should do exercise to make them stronger, but being an extreme 24/7 computer junkie and certain health issues make it very difficult to me. But I know, no fscking excuses about improving my health...
Any news about this?

**Comment by webczat** [comment]
from what i remember, it should be already done and fixed.
